certmgr/metrics/metrics.go

```diff
@@ -49,16 +49,16 @@ func genServeIndex(addr string) func(http.ResponseWriter, *http.Request) {
 // configured.
 func Start(addr, port string) {
 	if addr == "" || port == "" {
-		log.Warning("metrics: no prometheus address or port configured")
+		log.Warn().Msg("metrics: no prometheus address or port configured, metrics disabled")
```

Comment:
nit: We capitalize Prometheus in the message below but not here. Either way, but something something consistency, something something :) 

```suggestion
		log.Warn().Msg("metrics: no Prometheus address or port configured; metrics disabled")
```

Reply:
Good catch, I've tried to make everything lowercase for consistency